### PR TITLE
Add 3 scenarios for Calculator solution

### PR DESCRIPTION
**Pull request by jakubch1**

### CODE REVIEW

File: samples/Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+**Scenarios**
```
Comment (MarcoRossignoli) [inline samples/Calculator/README.md:1]: Should we briefly explain how the solution is composed?

File: samples/Calculator/scenarios/scenario02/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+git clone https://github.com/microsoft/codecoverage.git
+cd codecoverage/samples/Calculator/tests/Calculator.Core.Tests/
+dotnet test --collect "Code Coverage;Format=cobertura"
+```
+
+You can also use [run.ps1](run.ps1) to collect code coverage.
+
+**Collect code coverage inside github workflow**
+
+```shell
```
Comment (MarcoRossignoli) [inline samples/Calculator/scenarios/scenario02/README.md:17]: Wonder if we could have 2 link here one pointing github yaml and another azdo and the 2 files in the same folder of the scenario. In that files we can in future add different "way" or configuration, like how to move files around at the end etc...

File: samples/Calculator/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+1. [***Scenario 01*** Default settings (binary .coverage report)](scenarios/scenario01/README.md)
+2. [***Scenario 02*** Cobertura format by command line](scenarios/scenario02/README.md)
```
Comment (MarcoRossignoli) [inline samples/Calculator/README.md:4]: that's fine for now...maybe in future we need to organize the index per "context/area" but fine for now imo.

**Review by MarcoRossignoli — APPROVED** [review]
Some thoughts but LGTM

**Review by fhnaseer — APPROVED** [review]

File: samples/Calculator/README.md (hunk 1 above)
Comment (jakubch1) [reply samples/Calculator/README.md]: added

**Pull request merged**